packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+import fs from 'fs';
+import readline from 'readline';
+import { zip, unzip } from 'lodash';
```

Comment:
How about using Python's builtin zip/unzip? See https://docs.python.org/3/library/functions.html#zip.

Reply:
There is not 'unzip' in Python's builtin?
How to unzip, may use zip(*zip(x, y)):
```python
>>> x = [1, 2, 3]
>>> y = [4, 5, 6]
>>> zipped = zip(x, y)
>>> list(zipped)
[(1, 4), (2, 5), (3, 6)]
>>> x2, y2 = zip(*zip(x, y))
>>> x == list(x2) and y == list(y2)
True
```
'*' is useless in js.

Reply:
Okay, let's use lodash edition for now :(

Reply:
enen :(